UnitTests/BraintreeCoreTests/RepeatingTimer_Tests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+@testable import BraintreeCore
+
+class RepeatingTimerTests: XCTestCase {
+
+    func testTimer_resumesAndSuspends() {
```

Comment:
Nitpick - we try to name tests like this format: `test<MethodName>_{optional when<Condition>}_<ExpectedResult>`

This test might be more readable as two separate tests. For ex: `testResume_callsEventHandler` or `testSuspend_eventHandlerNotCalled`